packages/app/api-common/package.json

```diff
@@ -43,7 +43,7 @@
 		"prettier": "3.3.3",
 		"rimraf": "^5.0.5",
 		"typescript": "5.5.3",
-		"vite": "^5.3.3",
+		"vite": "5.3.4",
```

Comment:
This causes to have in node-modules a non-compatible version with the one on out vite package